apps/src/templates/SchoolInfoInputs.jsx

```diff
@@ -134,7 +134,6 @@ export default class SchoolInfoInputs extends Component {
          />
         }
         {(outsideUS || ncesInfoNotFound || noDropdownForSchoolType) &&
-         !(outsideUS && !askForName) &&
```

Comment:
We can just remove this guard outright, because `askForName` is already used just below to omit the name field if it's not needed:
```jsx
schoolName={askForName ? this.props.schoolName : SchoolNotFound.OMIT_FIELD}
```